cluster/sync.sh

```diff
@@ -6,45 +6,32 @@ cdi="${cdi##*/}"
 echo $cdi
 
 source ./hack/build/config.sh
+source ./hack/build/common.sh
 source ./cluster/gocli.sh
+source ./cluster/${KUBEVIRT_PROVIDER}/provider.sh
 
 CDI_NAMESPACE=${CDI_NAMESPACE:-cdi}
 
 # Set controller verbosity to 3 for functional tests.
 export VERBOSITY=3
 
 registry_port=$($gocli ports registry | tr -d '\r')
-registry=localhost:$registry_port
+registry=${IMAGE_REGISTRY:-localhost:$registry_port}
```

Comment:
I think this should be `DOCKER_REPO` not `IMAGE_REGISTRY`

Reply:
Oh!  Yes!  I was thinking about changing away from DOCKER_REPO and just use a generic image registry variable (ie quay.io etc).  I decided to do that as a follow up but didn't clean that last part out of there.

Reply:
I think I've got that fixed up at lest a little better than what I had.  Let me know if that looks ok to you.  We can always work out some things in more detail later too.